hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/AbstractRpcClient.java

```diff
@@ -165,12 +167,15 @@ public AbstractRpcClient(Configuration conf, String clusterId, SocketAddress loc
       MetricsConnection metrics) {
     this.userProvider = UserProvider.instantiate(conf);
     this.localAddr = localAddr;
-    this.tcpKeepAlive = conf.getBoolean("hbase.ipc.client.tcpkeepalive", true);
     this.clusterId = clusterId != null ? clusterId : HConstants.CLUSTER_ID_DEFAULT;
     this.failureSleep = conf.getLong(HConstants.HBASE_CLIENT_PAUSE,
       HConstants.DEFAULT_HBASE_CLIENT_PAUSE);
-    this.maxRetries = conf.getInt("hbase.ipc.client.connect.max.retries", 0);
-    this.tcpNoDelay = conf.getBoolean("hbase.ipc.client.tcpnodelay", true);
+    this.maxRetries = conf.getInt(CLIENT_CONNECT_MAX_RETRIES, 0);
+    this.tcpNoDelay = conf.getBoolean(CLIENT_TCP_NODELAY, true);
+    this.tcpKeepAlive = conf.getBoolean(CLIENT_TCP_KEEPALIVE, true);
+    this.bufferLowWatermark = conf.getInt(CLIENT_BUFFER_LOW_WATERMARK, DEFAULT_CLIENT_BUFFER_LOW_WATERMARK);
```

Comment:
This is only used for NettyRpcClient so better put this into the NettyRpcClient?

Reply:
> This is only used for NettyRpcClient so better put this into the NettyRpcClient?

Yeah,I referred to other constant of RpcClient like SOCKET_TIMEOUT_WRITE so that I put this into RpcClient.I will put these constants into NettyRpcClient.